src/Framework/Analyzer/src/DelegateEndpoints/DelegateEndpointFixer.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.AspNetCore.Analyzers.DelegateEndpoints;
+
+public partial class DelegateEndpointFixer : CodeFixProvider
```

Comment:
Sam's suggestion was to put code fixes in a different assembly since it depends on the workspaces assembly. Could we do that?

Reply:
Yep -- I recall this now.

@sharwell Can you sanity check that the packaging was done directly here?

Also, do they still need to be split out even if the CodeFixes don't take an explicit dependency on the `Microsoft.CodeAnalysis.Workspaces` assembly?

Reply:
> Also, do they still need to be split out even if the CodeFixes don't take an explicit dependency on the

CodeFixProvider is defined in Microsoft.CodeAnalysis.Workspaces, so it's impossible to avoid.